cmd/kops/get_instances.go

```diff
@@ -108,9 +108,11 @@ func RunGetInstances(ctx context.Context, f *util.Factory, out io.Writer, option
 		return err
 	}
 
+	emptyNodeName := "NotJoinedYet"
 	nodeList, err := k8sClient.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
 	if err != nil {
-		klog.V(2).Infof("error listing nodes: %v", err)
+		klog.Warningf("error listing nodes. Cannot determine node status: %v", err)
+		emptyNodeName = "APIUnavailable"
```

Comment:
NIT: Should we make this a bit more clear that it's the k8s API that is unavailable since we normally deal with both the k8s api and cloud provider API?

Reply:
Any suggestion on the wording?  don't think we can make the node name more verbose at least.